package.rb

```diff
@@ -0,0 +1,11 @@
+class Package
```

Comment:
well, when you start adding more files, then I suggest to create directories like `lib` and `bin` so it will be clear what is intented to run and what is only library for that scripts.